hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HsyncGenerator.java

```diff
@@ -53,32 +57,38 @@
     versionProvider = HddsVersionProvider.class,
     mixinStandardHelpOptions = true,
     showDefaultValues = true)
-public class HsyncGenerator extends HadoopNestedDirGenerator implements Callable<Void> {
+public class HsyncGenerator extends BaseFreonGenerator implements Callable<Void> {
   private static final Logger LOG = LoggerFactory.getLogger(HsyncGenerator.class);
 
   @CommandLine.ParentCommand
   private Freon freon;
 
+  @Option(names = {"--path"},
+      description = "Hadoop FS file system path. Use full path.",
+      defaultValue = "o3fs://bucket1.vol1")
+  private String rootPath;
+
   @Option(names = {"--bytes-per-write"},
       description = "Size of each write",
-      defaultValue = "1024")
+      defaultValue = "8")
   private int writeSize;
 
-  @Option(names = {"--number-of-files"},
-      description = "Number of files to run test.",
-      defaultValue = "1")
-  private int numberOfFiles;
+  @Option(names = {"--writes-per-transaction"},
+      description = "Size of each write",
+      defaultValue = "32")
+  private int writesPerTransaction;
```

Comment:
Could we also add this option to the command in the test to make sure it works correctly?